content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/about-github-classroom.md

```diff
@@ -0,0 +1,45 @@
+---
+title: About GitHub Classroom
+intro: '{% data variables.product.prodname_classroom %} is a space where teachers, organizations or school administrators can create assignments, collaborate with teaching assistants, and invite students in a single course.'
+type: overview
+topics: 
+  - Fundamentals
+versions:
+  fpt: '*'
+---
+
+
+## About {% data variables.product.prodname_classroom %}
+
+{% data variables.product.prodname_classroom %} is an all-in-one platform that makes it easier for teachers and school administrators to manage students' course-works, assignments and code submissions. With {% data variables.product.prodname_classroom %}, you can create group assignments and assign your students to teams, which makes it easier for your students to collaborate and for you and for you to grade them.
+
+With {% data variables.product.prodname_classroom %}, you can create template repositories with boilerplate code, documentation and other resources to serve as starter code for your students  on an assignment. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-assignment-from-a-template-repository)."
+
+You can also configure workflows, so that on each push of code to a pull request, your students' submissions are autograded. To learn more about autograding in GitHub Classroom, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/use-autograding)."
+
+{% data variables.product.prodname_classroom %} lets you enable pull request for feedback. This creates a special pull request titled **Feedback** in the assignment repository for each student or team. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/leave-feedback-with-pull-requests)."
+
+You can optionally connect a learning management system (LMS) with {% data variables.product.prodname_classroom %}. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/connect-a-learning-management-system-course-to-a-classroom)."
+
+
+## GitHub Classroom Features 
```

Comment:
```suggestion
## {% data variables.product.prodname_classroom %} features 
```

Small suggestion to use a variable

Reply:
Also, with the accessibility initiative, we add sentences between headings to improve the screenreader experience 🙂